crates/sui-framework/build.rs

```diff
@@ -50,8 +50,37 @@ fn main() {
 fn build_framework_and_stdlib(sui_framework_path: &Path, out_dir: PathBuf) {
     let config = MoveBuildConfig {
         generate_docs: true,
+        test_mode: false,
```

Comment:
Do we still need this?

Reply:
This is what allows us to avoid going all in - we have two versions of bundled binaries now, one with and one without test code included. Code version verification can pick one depending on whether `BuildConfig`s `test_mode` flag is set to `true` or not.

Reply:
I meant, it should already be set to `false` right? Or is there some path where it isn't?

Reply:
Aaah... You are right, it defaults to `false`, but I did feel like it wouldn't hurt clarity to have the exact counterparts in both cases, with this flag set to `false` and `true` explicitly. Happy to change if you feel like it has the opposite effect.

Reply:
maybe just a debug assert then? Feels like a good use for that